app/game/[gameId]/view/page.tsx

```diff
@@ -1,18 +1,40 @@
-import { getGame } from '../../index'
+"use client"
```

Comment:
Nit: I would rather move some of this client side logic to a new component and use that within this server side component, as opposed to making the entire thing client

Reply:
:0 ok, not entirely sure how to do this one, but I'll give it a shot!